{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -16,3 +16,19 @@ universal = 1
 
 [flake8]
 exclude = docs
+
+[aliases]
+{%- if cookiecutter.use_pytest == 'y' %}
+test = pytest
+{%- endif %}
+# Define setup.py command aliases here
+{%- if cookiecutter.use_pytest == 'y' %}
+
+[pytest]
+# Set permanent options for the "pytest" setup command here
```

Comment:
So, I was just trying this in a clean virtualenv and got the following warning:

```
============================================== warnings summary ===============================================
None
  [pytest] section in setup.cfg files is deprecated, use [tool:pytest] instead.

-- Docs: http://doc.pytest.org/en/latest/warnings.html
==================================== 2 passed, 1 warnings in 0.10 seconds =====================================
```

Does it make sense to have these two sections?
Sounds like we should use only `tools:pytest`, no?